hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -955,8 +953,8 @@ private ContainerTokenSecretManager createContainerTokenSecretManager(
       scmCertificateClient = new SCMCertificateClient(securityConfig,
           certSerialNumber, SCM_ROOT_CA_COMPONENT_NAME);
     }
-    return new ContainerTokenSecretManager(securityConfig,
-        expiryTime);
+    return new ContainerTokenSecretManager(expiryTime,
```

Comment:
Before this code piece, there is the regular checks for certificate lifetime, and existence. Do we still need those?

Reply:
I think the check you mention is in the [DefaultCertificateClient](https://github.com/duongnguyen0/ozone/blob/77bd133e0a899adb31c106de08484acc2d482d2d/hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/DefaultCertificateClient.java#L1181-L1181), and it's independent from the implementation of `ContainerTokenSecretManager`.